src/lib/makeParser.ts

```diff
@@ -26,6 +30,12 @@ export default function makeParser<T>(
 
     validate(clone)
     if (validate.errors) {
+      if (process.env.NODE_ENV !== 'production') {
+        // in dev mode the missing properties are not shown and this is very
+        // confusing. This will output any time there's an error parsing the env
+        // eslint-disable-next-line no-console
```

Comment:
**Question**

I'm not sure I quite follow this. Is the expectation from this that we'll always run with a `NODE_ENV` of `production`, even when developing locally?

Reply:
yeah good question. Help me phrase this explanation better.

In production, we do not need the `console.error`. The regular process even catchers (see [process.on](https://github.com/bloomberg/coffeechat/pull/20/files#diff-617e8f6a5c7c6aabbd617efcc6d688aa2a6d1ef17a51cae98724576503cea63bR11) ) capture the exception fine and print all the errors in the schema.

For whatever reason, in development (`NODE_ENV!==production`) it only prints something like `parseError`. However, `console log` does print the schema error. 

uhmn.. maybe we should leave the `console.error` always.  In production it would print the schema errors twice.